src/bundle.js

```diff
@@ -0,0 +1,46 @@
+import Mfs from 'memory-fs'
+import webpack from 'webpack'
```

Comment:
I agree with making it pluggable. I've got a substantial Nexe setup and I've been maintaining my own version because in v1 Browserify wasn't pluggable. I'm using Rollup for flat bundles and fast startup time.